x/swap/testutil/suite.go

```diff
@@ -103,15 +104,15 @@ func (suite *Suite) NewAccountFromAddr(addr sdk.AccAddress, balance sdk.Coins) a
 // CreateVestingAccount creates a new vesting account from the provided balance and vesting balance
 func (suite *Suite) CreateVestingAccount(initialBalance sdk.Coins, vestingBalance sdk.Coins) authtypes.AccountI {
 	acc := suite.CreateAccount(initialBalance)
-	bacc := acc.(*authtypes.BaseAccount)
+	bacc := acc.(*ethermint.EthAccount)
```

Comment:
unlikely we will change the base account to EthAccount but it's easier to keep it for now during development